code/modules/food&drinks/kitchen machinery/monkeyrecycler.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/monkey_recycler
 	name = "monkey recycler"
-	desc = "A machine used for recycling dead monkeys into monkey cubes. It requires 5 monkeys per cube."
+	desc = "A machine used for recycling dead monkeys into monkey cubes. It requires [required_grind] monkeys per cube."
```

Comment:
If you compile you will know you can't do this: you can't set default values with non constants. Including a var in the default value will not compile.